packages/actor-init-sparql/README.md

```diff
@@ -87,10 +87,14 @@ Use `bin/query-dynamic.js` when running dynamically inside the Comunica monorepo
 Start a webservice exposing http://fragments.dbpedia.org/2015-10/en via the SPARQL protocol, i.e., a _SPARQL endpoint_.
 
 ```bash
-$ comunica-sparql-http "{ \"sources\": [{ \"value\" : \"http://fragments.dbpedia.org/2015/en\" }]}"
+$ comunica-sparql-http "{ \"sources\": [\"http://fragments.dbpedia.org/2015/en\"]}"
 ```
 
-_(Use `\"type\": \"file\"` if you want to query over remote RDF files)_
+or
+
+```bash
+$ comunica-sparql-http "{ \"sources\": [{ \"type\" : \"file\", \"value\" : \"https://ruben.verborgh.org/profile/\" }]}"
```

Comment:
What was the reason the type was kept here, and why was it written like this and not `file@http://...` or is support for those formats not added?

Reply:
This was added to clarify that a type can be forced on a source here as well, just like above.

The `file@http://...` shorthand is not (yet) supported in `comunica-sparql-http`.

Reply:
Wouldn't it only require a change in the getDataSourceType and getDataSourceValue functions or is there more to it than that?

Reply:
The shorthand you mean? Sure, that wouldn't be that much work, but that's not in scope of this PR. See https://github.com/comunica/comunica/issues/433